ouroboros-consensus/test-storage/Test/Ouroboros/Storage/VolatileDB/StateMachine.hs

```diff
@@ -247,10 +250,12 @@ runPure dbm (CmdErr cmd err) =
         go = case cmd of
             GetBlock bid       -> do
                 Blob <$> getBlockModel tnc bid
+            GetHeader bid ->
+                Blob <$> getHeaderModel tnc bid
             GetBlockIds        ->
                 Blocks <$> getBlockIdsModel tnc
-            PutBlock b pb      ->
-                Unit <$> putBlockModel tnc err (BlockInfo b (guessSlot b) pb) (BL.lazyByteString $ toBinary (b, pb))
+            PutBlock b pb h    ->
+                Unit <$> putBlockModel tnc err (BlockInfo b (guessSlot b) pb (fromIntegral headerOffset) 1) (BL.lazyByteString $ toBinary (b, pb, h))
```

Comment:
This magic 1 should be fixed, as mentioned before.
Also, this line is getting rather long.